core/esmf-aspect-meta-model-python/tests/integration/test_characteristics.py

```diff
@@ -22,6 +22,7 @@
     Measurement,
     Quantifiable,
     StructuredValue,
+    State
```

Comment:
Apparently the code style check fails, because this needs to be sorted lexicographically:
```suggestion
    State,
    StructuredValue
```